LiveSplit.AutoSplitters.xml

```diff
@@ -1,5 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <AutoSplitters>
+    <AutoSplitter>
+        <Games>
+            <Game>DragonFangZ - The Rose & Dungeon of Time</Game>
```

Comment:
XML does not accept a `&` character because it is a special character.
```suggestion
            <Game>DragonFangZ - The Rose &amp; Dungeon of Time</Game>
```